<ui version="4.0" >
 <class>CreateUser</class>
 <widget class="QWidget" name="CreateUser" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>70</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>Last:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userName" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>260</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>Enter Name</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="saveCancel" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>310</y>
     <width>166</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstName" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>90</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastName" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>90</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="telephone" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask" >
    <string>DDDDDDDDDD; </string>
   </property>
   <property name="maxLength" >
    <number>10</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="unit" >
   <property name="geometry" >
    <rect>
     <x>460</x>
     <y>160</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>240</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>Move in Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>240</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>260</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="moveInDate" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>260</y>
     <width>110</width>
     <height>29</height>
    </rect>
   </property>
   <property name="buttonSymbols" >
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="date" >
    <date>
     <year>2010</year>
     <month>3</month>
     <day>4</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_9" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>240</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>Password</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="privateTele" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>190</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Private Tele?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>460</x>
     <y>120</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Unit to 
move  into</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>20</y>
     <width>191</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>191</width>
     <height>0</height>
    </size>
   </property>
   <property name="font" >
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text" >
    <string>CREATE MEMBER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>70</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>First</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>130</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>Pre-Coop Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>130</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>Tele. </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pastAddress" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>150</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="over21" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>310</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Over 21yrs old?</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
